recipes/pyclustering/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set name = "pyclustering" %}
+{% set version = "0.9.2" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8271610cd71f0b45bf185c4e478768fed64d6105153b3c3e5617a4711a4ae6b3
+
+build:
+  number: 0
+  # detect_binary_files_with_prefix: False
+  # binary_relocation:
+  #   - 'lib/*.so'
+  # ignore_run_exports:        # Cf comment above
+  #   - {{ compiler('cxx') }}
+  #   - {{ compiler('c') }}
+  #   - core
+  missing_dso_whitelist:
+    - /lib64/libm.so.6  # [linux]
+    - /lib64/libstdc++.so.6  # [linux]
+    - /lib64/ld-linux-x86-64.so.2  # [linux]
+    - /lib64/libgcc_s.so.1  # [linux]
+    - /lib64/libc.so.6  # [linux]
+    - /lib64/libSystem.B.dylib  # [osx]
+    - /lib64/ld-linux.so.2  # [linux]
```

The missing_dso_whitelist operates as a string match.  If $RPATH/blah is what conda-build is complaining about, you can try 
```suggestion
    - $RPATH/ld-linux.so.2  # [linux]
```

---

@msarahan thanks, trying that!